index.js

```diff
@@ -1,11 +1,8 @@
-/*jshint node:true */
-
 'use strict';
 
 var gutil = require('gulp-util');
-var c = gutil.colors;
 var error = gutil.PluginError;
-var es = require('event-stream');
+var through = require('through2');
```

Comment:
`thourgh@2.0.0` was released only a couple of months ago, unsure if it's still an issue? I have no trouble changing it, but I need a callback after all files are processed. If `readable-stream` has that, then no problem :smile: